src/python/elrs_helpers.py

```diff
@@ -46,18 +13,17 @@ def get_git_version(env):
     ver = "ver.unknown"
     sha = "000000"
 
-    git_repo = get_git(env)
-    if git_repo:
-        import git
-        sha = git_repo.head.object.hexsha
+    try:
+        call_git = lambda cmd: subprocess.check_output(["git"] + cmd).decode("utf-8").rstrip('\r\n')
+        sha = call_git(["symbolic-ref", "-q", "--short", "HEAD"])
+        ver = call_git(["rev-parse", "--abbrev-ref", "HEAD"])
```

Comment:
Should this be moved into the exception handler below?

Reply:
No, I don't think so - or is there a situation where we are not on any branch?

Reply:
However, I think it is redundant, right? `git describe --all --exact-match` will either print `heads/current-branch` or `tags/current-tag`. So whenever `rev-parse --abbrev-ref HEAD` is able to pickup a branch, describe will also be able to, right?

Reply:
It's not redundant. What you have will work actually. Sometimes we do checkout unnamed commits when doing git-bisect to find where errors are introduced.

Reply:
Ok, I reintroduced it.